ci-operator/step-registry/shiftstack/liveliness/shiftstack-liveliness-commands.sh

```diff
@@ -8,6 +8,12 @@ CLUSTER_NAME=$(<"${SHARED_DIR}/CLUSTER_NAME")
 OPENSTACK_EXTERNAL_NETWORK="${OPENSTACK_EXTERNAL_NETWORK:-$(<"${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK")}"
 # Recycling BASTION_FLAVOR as it's a small flavor we can re-use.
 TESTING_FLAVOR="${TESTING_FLAVOR:-$(<"${SHARED_DIR}/BASTION_FLAVOR")}"
+ZONES="${ZONES:-$(<"${SHARED_DIR}/ZONES")}"
+
+AZ_ARG=""
+if [ -n "${ZONES}" ]; then
+  AZ_ARG="-a ${ZONES}"
```

Comment:
What happens when we have multiple zones? Right now, it looks like it's going to explode in server.sh (based on what is currently in https://github.com/shiftstack/shiftstack-ci/pull/210/).
Should we spin a VM in each zone?

Reply:
for now let's just use the first zone in the list. Our CI clouds are not multi zones.